chapters/en/slides/module2/module2_03.Rmd

```diff
@@ -0,0 +1,284 @@
+We don't need to specify a column name for the y-axis,
+since we are just counting values in each categorical group on the y-axis.
+
+---
+
+## Ordering bars makes them easier to interpret
+
+```{python}
+alt.Chart(gm2018).mark_bar().encode(
+    x='count()',
+    y=alt.Y('region'))
+```
+
+Notes:
+Unless the categorical axis has a natural order to it,
+it is best to sort bar charts from high to low values.
+This makes it easier to see any trends in the data,
+and to compare bars of similar height more carefully.
+
+To sort the bars,
+we will use the helper functions `alt.X` and `alt.Y`,
+instead of `x=` and `y=`.
+These have the role of customizing things like order, groups, and scale for the axis.
+When using just `y='column'`,
+we're still calling `alt.Y()` under the hood,
+we just save ourselves some typing.
+
+In this slide,
+we use `alt.Y`
+on its own without any additional argument
+to recreate the same plot as above
+just to demonstrate how it works.
+
+---
+
+## Ordering bars makes them easier to interpret
+
+```{python}
+alt.Chart(gm2018).mark_bar().encode(
+    x='count()',
+    y=alt.Y('region', sort='x'))
+```
+
+Notes:
+Sorting can now be done via the `sort` parameter in `alt.Y`.
+In this example,
+we're typing `sort='x'` to specify that we want to sort
+according to the values on the y-axis.
+
+When sorting by value,
+it is often more visually appealing with the longest bar
+the closet to the axis line,
+as in this slide.
+
+---
+
+## The longest bar should be the closest to the axis line
+
+```{python}
+alt.Chart(gm2018).mark_bar().encode(
+    x='count()',
+    y=alt.Y('region', sort='-x'))
+```
+
+If we for some reason would want the reverse order,
+we could use the minus sign before the axis reference.
+
+---
+
+## Sorting in a custom order
+
+```{python}
+my_order = ['Africa', 'Europe', 'Oceania', 'Asia', 'Americas']
+alt.Chart(gm2018).mark_bar().encode(
+    x='count()',
+    y=alt.Y('region', sort=my_order))
+```
+
+Notes:
+Sometimes there is a natural order to our data
+that we want to use for the bars,
+for example days of the week.
+
+For situations like this,
+we can pass a list or array
+to the `sort` parameter.
+
+We can either create this list manually as in thie slide,
+or use the pandas sort function if we wanted
+for example reverse alphabetical order.
+
+To learn more about good considerations
+when plotting counts of categorical observations,
+you can refer to [chapter 6 of Fundamental of Data Visualizations](https://clauswilke.com/dataviz/visualizing-amounts.html).
+
+---
+
+## Using bars to show counts of quantitative values
+
+```{python}
+alt.Chart(gm2018).mark_bar().encode(
+    x=alt.X('life_expectancy'),
+    y='count()')
+```
+
+Notes:
+So far we have been counting observations in categorical groups
+(the continents).
+Another common use of bar charts is to count how many observations
+fall into numerical intervals
+in an attempt to estimate the distribution, or shape, of the entire dataset.
```

Comment:
I think a better way to describe the goal here is to understand the variation in our measurements. Specifically to know the range and frequency of the observed values (which is a definition for a distribution).

Reply:
I wasn't completely happy with the word "shape" wither, so I'll make sure to specify exactly what it entails as you suggested.